crnk-core/src/main/java/io/crnk/core/engine/transaction/TransactionRunner.java

```diff
@@ -4,5 +4,5 @@
 
 public interface TransactionRunner {
 
-	<T> T doInTransaction(Callable<T> callable);
+	<T> T doInTransaction(Callable<T> callable) throws Exception;
```

Comment:
not sure about this, can you not just catch and rethrow RuntimeExceptions and wrap the very rare checked exceptions into a runtime exception? because this change is slightly breaking.

Reply:
yeah :-/ I think I'll give RuntimeException a try